dipy/data/__init__.py

```diff
@@ -362,6 +362,10 @@ def mrtrix_spherical_functions():
 
 def get_cmap(name):
     """Makes a callable, similar to maptlotlib.pyplot.get_cmap"""
+    if name.lower() == "accent":
+        warnings.warn("The `Accent` colormap is deprecated as of version" +
```

Comment:
Oh yes - just what I needed! 

Also adding a test for the color-map for MPL < 2!